internal/controllers/machine_controller.go

```diff
@@ -244,6 +252,9 @@ func (r *MachineReconciler) startCheckAndEnqueueVolumeResize(ctx context.Context
 				}
 
 				if lastVolumeSize := getLastVolumeSize(machine, GetUniqueVolumeName(plugin.Name(), volumeID)); volumeSize != ptr.Deref(lastVolumeSize, 0) {
+					if eventErr := r.eventStore.AddEvent(machine.Metadata, corev1.EventTypeNormal, "SizeChangedVolume", fmt.Sprintf("Volume size changed %s. lastVolumeSize: %d. volumeSize: %d .", volume.Name, *lastVolumeSize, volumeSize)); eventErr != nil {
```

Comment:
why . and you have different space for .

Reply:
I referred to a Kubernetes document where the message ended with a period ("."). However, this seems to apply only to that specific object. I tested it for a pod, and the message doesn't end with a period. I will remove the period.

Reply:
removed